jquery.simulate.js

```diff
@@ -200,8 +200,8 @@ $.extend( $.simulate.prototype, {
 		return event;
 	},
 
-	dispatchEvent: function( elem, type, event ) {
-		if ( elem[ type ] ) {
+	dispatchEvent: function( elem, type, event, options ) {
+		if ( elem[ type ] && !options ) {
```

Comment:
Ah, you're right about that. Definitely needs to me more granular (i.e. options only apply to mouse and key events)